nat-lab/tests/test_dns.py

```diff
@@ -28,6 +28,7 @@
         pytest.param(DNS_IPV6_SERVER_ADDRESS, id="IPv6"),
     ],
 )
+@pytest.mark.xfail(reason="Test is flaky - JIRA issue LLT-4242")
```

Comment:
Maybe I overlooked something, but I see this test failing only once and only for IPv6 setting, so maybe we could mark only IPv6 part, or even wait a bit if this failure reoccurs?

Reply:
We could see this entire week and then decide ?